src/main/java/plugily/projects/villagedefense/arena/managers/spawner/SimpleZombieSpawner.java

```diff
@@ -0,0 +1,144 @@
+package plugily.projects.villagedefense.arena.managers.spawner;
+
+import java.util.Random;
+import org.bukkit.Location;
+import org.bukkit.entity.Zombie;
+import plugily.projects.villagedefense.arena.Arena;
+import plugily.projects.villagedefense.arena.options.ArenaOption;
+import plugily.projects.villagedefense.creatures.CreatureUtils;
+
+/**
+ * The interface for simple zombie spawner
+ */
+public interface SimpleZombieSpawner extends ZombieSpawner {
+  /**
+   * Get the minimum wave to spawn the zombies
+   *
+   * @return the wave
+   */
+  int getMinWave();
```

Comment:
I think a default method would be better here, so the minimum wave number is 1. After this you can remove from implementations where the min wave is 1.